### PR TITLE
SPOI-10717 Configuration Packages Page

### PR DESCRIPTION
**Pull request by peyao**

### CODE REVIEW

**Comment by peyao** [comment]
Updated with a new section called **Exporting and Importing Application Configurations** since PR first opened.

File: mkdocs.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ pages:
     - Apex Development Setup: apex_development_setup.md
     - Generate New Project in IDE: configure_IDE.md
     - Applications: application_development.md
+    - Application Configurations: application_configurations.md
```
Comment (sashadt) [inline mkdocs.yml]: Let's move this into Platform section below RTS.

File: docs/application_configurations.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+Application Configurations
+========================
+
+This guide will walk you through the steps in creating, modifying, and launching
```
Comment (sashadt) [inline docs/application_configurations.md]: This sentence can be removed.

File: docs/application_configurations.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+=====================================
+
+  1. Navigate to either the *Applications* page or the *Application Configurations*
+  page in the **Develop** section.
+  2. If you're on the  *Applications* page, click the **configure** button 
+  next to the Application you want to configure, and then click **create new configuration**.
+    ![](images/application_configurations/image00.png)
+  Or if you're on the *Application Configurations* page, click on the **create new** button.
+    ![](images/application_configurations/image01.png)
+  3. Select the **Source Application** from the dropdown, choose a
```
Comment (sashadt) [inline docs/application_configurations.md]: Mention that application being configured is selected by default.

**Pull request merged**